app/models/request.rb

```diff
@@ -0,0 +1,73 @@
+class Request < ActiveRecord::Base
+  belongs_to :requestee, :class_name => "User"
+  belongs_to :requester, :class_name => "User"
+
+  validates_presence_of :requestee
+  validates_presence_of :requester
+  validates_presence_of :item_type
+  validates_inclusion_of :item_type, in: ["Movie", "AndroidApp"]
+  validates_presence_of :status
+  validates_inclusion_of :status, in: ["pending", "sent", "seen", "successful"]
+
+  validates_uniqueness_of :item_type, scope: [:requestee, :requester],
+    conditions: -> { where( status: ["pending","sent"] ) }
+
+  class RequestValidator < ActiveModel::Validator
```

Comment:
It would later turn out to have it's own properties different than `Recommendation`. We shouldn't mix up things imo. Besides, we need to track request history as well. Read slack.